app/views/layouts/_base.html.erb

```diff
@@ -11,6 +11,7 @@
   <meta name="x-section-name" content="Business" />
   <meta name="x-section-link" content="/browse/business" />
   <meta name="x-section-format" content="trade-tariff" />
+  <meta name="description" content="<%= I18n.t('meta_description') %>">
```

Comment:
It would be better to pull this from the artefact given we've already loaded it from contentapi.  That'll avoid having to hard-code this here.